src/code/z_room.c

```diff
@@ -87,7 +87,7 @@ s32 Room_StartRoomTransition(PlayState* play, RoomContext* roomCtx, s32 index) {
         size = play->roomList[index].vromEnd - play->roomList[index].vromStart;
         roomCtx->activeRoomVram =
             (void*)(((s32)roomCtx->roomMemPages[roomCtx->activeMemPage] - (size + 8) * roomCtx->activeMemPage + 8) &
-                    0xfffffff0);
+                    0xFFFFFFF0);
```

Comment:
This is an ALIGN16

Reply:
I think it's missing the `+ 0xF` to be an `ALIGN16`, but I can change it to `~0xF`

Reply:
OoT has
```c
roomCtx->unk_34 = (void*)ALIGN16((u32)roomCtx->bufPtrs[roomCtx->unk_30] - ((size + 8) * roomCtx->unk_30 + 7));
```

Reply:
Ah I see. Has to be `- 7` but same concept. Yeah it works

Reply:
Strange that it's different at all, will be interesting to compare the retail asm.